js/modules/types/errors.js

```diff
@@ -0,0 +1,9 @@
+const ensureError = require('ensure-error');
+
+const Privacy = require('../privacy');
+
+//      toLogFormat :: Error -> String
+exports.toLogFormat = (error) => {
+  const normalizedError = ensureError(error);
+  return Privacy.redactAll(normalizedError.stack);
```

Comment:
Do we need to redact here, if we're already redacting in `logging.js`?

Reply:
Sure; I must have added this before the general redaction in the logging module.